tests/Http/Profile/EditTest.php

```diff
@@ -346,6 +348,29 @@
     expect($user->refresh()->prefers_anonymous_questions)->toBeFalse();
 });
 
+test('default_tab can be updated', function () {
```

Comment:
no need for extra test just add it into `'test('profile information can be updated')`